python/processors/ahadd.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env python
```

Comment:
How is this script comparing to the haddnano below? It seems both are just for hadd.